extension-manifest-v3/src/background/telemetry/metrics.js

```diff
@@ -390,7 +379,7 @@ class Metrics {
         this.storage[`${type}_${frequency}`] = timeNow;
         this.saveStorage(this.storage);
```

Comment:
Here, we are missing an `await`. IMHO, better to send too few messages than too many.